docker-compose.yml

```diff
@@ -827,6 +827,22 @@ services:
         max-size: "${DEFAULT_LOG_SIZE}"
         max-file: "${HIGH_PRIO_LOG_FILES}"
 
+  #
+  # FAF Solr standalone server
+  #
+  faf-solr:
+    # version: '3'
+    container_name: faf-solr
+    image: solr:8
```

Comment:
We want to pin the image version to a release family avoid unexpected breaking changes. Therefore please use `8.7-slim` (slim is for alpine images).